dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ESIndexUtilTest.java

```diff
@@ -0,0 +1,199 @@
+package com.dotcms.content.elasticsearch.business;
+
+import com.dotcms.content.elasticsearch.util.RestHighLevelClientProvider;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.rainerhahnekamp.sneakythrow.Sneaky;
+import org.elasticsearch.ElasticsearchStatusException;
+import org.elasticsearch.action.admin.indices.settings.put.UpdateSettingsRequest;
+import org.elasticsearch.action.support.master.AcknowledgedResponse;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.common.settings.Settings;
+import org.junit.BeforeClass;
+import com.dotcms.util.IntegrationTestInitService;
+import org.junit.Test;
+
+import static org.jgroups.util.Util.assertEquals;
+
+public class ESIndexUtilTest {
+
+    @BeforeClass
+    public static void prepare () throws Exception {
+        //Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    /**
+     * Method to Test: {@link ESIndexUtil#isReadOnly(String...)}
+     * When: The Index is not read only
+     * Should:return false
+     *
+     * @throws DotDataException
+     */
+    @Test
+    public void shouldReturnFalseIfTheCurrentIndicesAreReadOnly() throws DotDataException {
+        final IndiciesInfo indiciesInfo = APILocator.getIndiciesAPI().loadIndicies();
+
+        try {
+            putReadOnly(indiciesInfo.getWorking(), false);
+
+            final boolean readOnly = ESIndexUtil.isReadOnly(indiciesInfo.getWorking());
+
+            assertEquals(false, readOnly);
+        } finally {
+            putReadOnly(indiciesInfo.getWorking(), false);
+        }
+    }
+
+    /**
+     * Method to Test: {@link ESIndexUtil#isReadOnly(String...)}
+     * When: The Index is read only
+     * Should:return true
+     *
+     * @throws DotDataException
+     */
+    @Test
+    public void shouldReturnTrueIfTheCurrentIndicesAreReadOnly() throws DotDataException {
+        final IndiciesInfo indiciesInfo = APILocator.getIndiciesAPI().loadIndicies();
+
+        try {
+            putReadOnly(indiciesInfo.getWorking(), true);
+
+            final boolean readOnly = ESIndexUtil.isReadOnly(indiciesInfo.getWorking());
+
+            assertEquals(true, readOnly);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use assertTrue(x)/assertFalse(x) instead of assertEquals(true, x)/assertEquals(false, x) or assertEquals(Boolean.TRUE, x)/assertEquals(Boolean.FALSE, x).](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5642172)